build-linux

```diff
@@ -38,7 +38,6 @@ cd ..
 # parallel
 git clone https://github.com/krakjoe/parallel.git
```

Comment:
I've forked it to https://github.com/manticoresoftware/krakjoe-parallel . Let's use that instead.